QR-code-Genrator/README.md

```diff
@@ -0,0 +1,10 @@
+certifi==2020.12.5
```

Comment:
Why readme file has only requirements? Rename this file to requirements.txt and create a separate readme file. If unsure what to add in the readme, check merged scripts readme or copy this readme template: 

https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md

Reply:
@Goheljay Also, there are a lot of unnecessary modules. Remove them from the requirements.txt